File: Feitos em C Sharp/1009.cs
using System;
using System.Globalization;
namespace ConsoleApp7 {
    class Program {
        static void Main(string[] args) {
            string nome = Console.ReadLine();
            double a = double.Parse(Console.ReadLine());
            double b = double.Parse(Console.ReadLine());
            Console.WriteLine("TOTAL = R$ " + (a + (b * 0.15)).ToString("F2",CultureInfo.InvariantCulture));
        }
    }
}

File: Feitos em C/1009.c
#include <stdio.h>
int main() {
	char nome[30];
    double a=0,b=0;
    scanf("%s",&nome);
    scanf("%lf",&a);
    scanf("%lf",&b);
    printf("TOTAL = R$ %.2f\n",a+(b*.15));
	return 0;
}
